deploy/crds/logging.openshift.io_clusterlogforwarders_crd.yaml

```diff
@@ -0,0 +1,203 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: clusterlogforwarders.logging.openshift.io
+spec:
+  group: logging.openshift.io
+  names:
+    kind: ClusterLogForwarder
+    listKind: ClusterLogForwarderList
+    plural: clusterlogforwarders
+    singular: clusterlogforwarder
+  scope: cluster
+  subresources:
+    status: {}
+  validation:
+    openAPIV3Schema:
+      description: ClusterLogForwarder is the schema for the `clusterlogforwarder`
+        API.
+      properties:
+        apiVersion:
+          description: 'APIVersion defines the versioned schema of this representation
+            of an object. Servers should convert recognized schemas to the latest
+            internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#resources'
+          type: string
+        kind:
+          description: 'Kind is a string value representing the REST resource this
+            object represents. Servers may infer this from the endpoint the client
+            submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#types-kinds'
+          type: string
+        metadata:
+          type: object
+        spec:
+          description: ClusterLogForwarderSpec defines the desired state of ClusterLogForwarder
+          properties:
+            outputs:
+              description: Outputs are named output destinations for log messages.
+              items:
+                description: Output defines a destination for log messages.
+                properties:
+                  destination:
+                    description: "Destination of log messages, an absolute URL (must
+                      have a scheme). \n Valid URL schemes depend on `type`.  Special
+                      schemes 'tcp', 'udp' and 'tls' are used for output types that
+                      don't define their own URL scheme. Example: \n     { type: syslog,
+                      destination: tls://syslog.example.com:1234 } \n TLS with server
+                      authentication is enabled by the URL scheme, for example 'tls'
+                      or 'https'.  See `secretRef` for TLS client authentication."
+                    type: string
+                  insecure:
+                    description: Insecure must be true for intentionally insecure
+                      outputs. Has no function other than a marker to help avoid configuration
+                      mistakes.
+                    type: boolean
+                  name:
+                    description: Name used to refer to the output from a `pipeline`.
+                    type: string
+                  reconnect:
```

Comment:
I believe we can, but I've removed for now to be added when implemented. It may be we can just implement a default reconnect strategy that satisfies most cases, and leave anything more complicated to the specifics.